src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/IslHelper.groovy

```diff
@@ -137,6 +148,21 @@ class IslHelper {
                 isl.dstPort, newMaxBandwidth)
     }
 
```

Comment:
The rest of the new methods were migrated and slightly modified from PathHelper (we interact with the path by using Isls path representation and it seems obvious to move all these interactions to IslHelper).